pull_request_template.md

```diff
@@ -1,9 +1,17 @@
 **IMPORTANT: Please read the README before submitting pull requests for this project. Additionally, if your PR closes any open GitHub issues, make sure you include _Closes #XXXX_ in your comment or use the option on the PR's sidebar to add related issues to auto-close the issue that your PR fixes. **
 
-**Description:**
+### Jira Ticket 
+<!--- Add a link to the Jira ticket associated with this PR -->
 
-Please include a summary of the change and which issue is fixed or which feature is introduced. If changes to the behavior are made, clearly describe what changes.
-If changes to the UI are made, please include screenshots of the before and after.
+### Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- Describe the approach to solve the problem or complete the task -->
+    
+### QA and Testing Instructions
+<!--- Include any relevant details about testing and QA steps to be followed -->
```

Comment:
How does this sound? Any better or clearer?
`List the steps required to review the changes in locally or in staging`